app/models/async_transaction/vet360/base.rb

```diff
@@ -72,12 +79,41 @@ def self.update_transaction_from_api(transaction_record, api_response)
         transaction_record
       end
 
-      # Returns true if a transaction is "over"
-      # @return [Boolean]
+      # Returns true or false if a transaction is "over"
+      # @return [Boolean] true if status is "over"
       def finished?
         # These SHOULD go hand-in-hand...
         FINAL_STATUSES.include?(transaction_status) || status == COMPLETED
       end
+
+      # Wrapper for .refresh_transaction_status which finds any outstanding transactions
+      #   for a user and refreshes them
+      # @param user [User] the user whose transactions we're checking
+      # @params service [Vet360::ContactInformation::Service] an initialized vet360 client
+      # @return [Array] An array with any outstanding transactions refreshed. Empty if none.
+      def self.refresh_transaction_statuses(user, service)
+        ongoing_transactions = last_ongoing_transactions_for_user(user)
+        refreshed_transactions = []
+        ongoing_transactions.each do |transaction|
```

Comment:
Instead of the variable assignment of `ongoing_transactions` on line 95, you can just call this method directly here.  i.e.:

```ruby
last_ongoing_transactions_for_user(user).each do |transaction|
```